### PR TITLE
update NEWS.md and version.py for 2.1.1 release

### PR DESCRIPTION
**Pull request by Jasha10**
Diff for 2.1.1 release

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -1,3 +1,13 @@
+## 2.1.1 (2021-08-16)
+### Features
+
+- Add a throw_on_missing keyword argument to the signature of OmegaConf.to_container, which controls whether MissingMandatoryValue exceptions are raised. ([#501](https://github.com/omry/omegaconf/issues/501))
+
+### Miscellaneous changes
+
+- Fix a packaging issue (missing sdist dependency) ([#772](https://github.com/omry/omegaconf/issues/772))
```
Comment (omry) [inline NEWS.md:8]: I thought we had two misc changes.
sdist and the yaml dependency change.

Comment (Jasha10) [reply NEWS.md:9]: Yes, you're right. I've pushed an update to the NEWS file in https://github.com/omry/omegaconf/pull/778/commits/56ff6e5225b9685382f23b0faa4409f3617c9bf2.

Comment (omry) [reply NEWS.md:9]: just write it down in the news file directly, I suspect we never wrote a news fragment when fixing the version info.

Comment (Jasha10) [reply NEWS.md:9]: Yup, it's done. I used towncrier to make sure that the formatting was consistent, but I did not commit a news fragment.

Comment (Jasha10) [reply NEWS.md:9]: Here's the diff:
```text
diff --git a/NEWS.md b/NEWS.md
index 4de7528..6a1234a 100644
--- a/NEWS.md
+++ b/NEWS.md
@@ -1,10 +1,11 @@
-## 2.1.1 (2021-08-16)
+## 2.1.1 (2021-08-17)
 ### Features

 - Add a throw_on_missing keyword argument to the signature of OmegaConf.to_container, which controls whether MissingMandatoryValue exceptions are raised. ([#501](https://github.com/omry/omegaconf/issues/501))

 ### Miscellaneous changes

+- Update pyyaml dependency specification for compatibility with PEP440 ([#758](https://github.com/omry/omegaconf/issues/758))
 - Fix a packaging issue (missing sdist dependency) ([#772](https://github.com/omry/omegaconf/issues/772))
 - ```

**Review by omry — APPROVED** [review]

**Pull request merged**